autoload/denops/plugin.vim

```diff
@@ -174,7 +178,10 @@ function! s:DenopsSystemPluginPost() abort
     " behavior.
     let l:callbacks = has('nvim') ? l:callbacks : reverse(l:callbacks)
     for l:Callback in l:callbacks
-      call timer_start(0, { -> l:Callback() })
+      " Split function to create new callstack independent for loop,
+      " because overwrited callback references when
+      " before timer processing timings.
```

Comment:
Remove this comment while Vim script always parse lines. Or, you can move the comment to outside of `s:delay_callback` function.